ivy/functional/ivy/experimental/losses.py

```diff
@@ -93,3 +94,70 @@ def log_poisson_loss(
         return ivy.mean(loss, axis=axis, out=out)
     else:
         return ivy.inplace_update(out, loss) if out is not None else loss
+
+
+@handle_exceptions
+@handle_nestable
+@handle_array_like_without_promotion
+@inputs_to_ivy_arrays
+@handle_array_function
+def huber_loss(
+    y_true: Union[ivy.Array, ivy.NativeArray],
+    y_pred: Union[ivy.Array, ivy.NativeArray],
```

Comment:
```suggestion
    true: Union[ivy.Array, ivy.NativeArray],
    pred: Union[ivy.Array, ivy.NativeArray],
```
To be consistent with other loss functions.